substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1865,6 +1865,24 @@ pub mod pallet {
 		MinBalanceDeficitAdjusted { pool_id: PoolId, amount: BalanceOf<T> },
 		/// Claimed excess frozen ED of af the reward pool.
 		MinBalanceExcessAdjusted { pool_id: PoolId, amount: BalanceOf<T> },
+		/// A pool member's claim permission has been updated.
+		MemberClaimPermissionUpdated { member: T::AccountId, permission: ClaimPermission },
+		/// A pool's metadata was updated.
+		MetadataUpdated { pool_id: PoolId, caller: T::AccountId },
```

Comment:
Is adding `caller` useful here? Nothing against it, just trying to understand the rationale. Same for other events.

Reply:
In the case of `set_metadata`, both the pool's root and its bouncer can make that call - this is to disambiguate which did it, in case the two roles are performed by different accounts.

The same goes for the other calls: `nominate/chill` can be called by the pool's root/nominator.